### PR TITLE
Update versionstrings

### PR DESCRIPTION
**Pull request by seanog8**
I scanned the following firmwares. Emba is now able to detect the versions that are below each firmware and to find CVE-Entries for them.

Firmwares and Versionstrings:

re505:
https://www.tp-link.com/us/support/download/re505x/#Firmware
flashcp (mtd-utils) 2.0.2
ethswctl.c:v0.0.2 (January 27, 2009)
Dropbear v2016.74

FRITZ.Box_4040-07.26-88509-LabBETA.image:
https://avm.de/fritz-labor/frisch-aus-der-entwicklung/frisch-aus-der-entwicklung/
ftpd (GNU inetutils) 1.4.2
(GNU gettext-runtime) 0.19.8
chronyc (chrony) version 3.5 (-READLINE -SECHASH +IPV6 -DEBUG)

TL-WA120:
https://www.tp-link.com/us/support/download/tl-wa1201/#Firmware
conntrack v1.0.0 (conntrack-tools)

re650:
https://www.tp-link.com/us/support/download/re650/#Firmware
Dropbear multi-purpose version 2012.55

Trident_V7209_5:
https://www.manula.com/manuals/tridentrfid/software-and-firmware/1/en/topic/router-firmware
pppd version 2.4.2

### CODE REVIEW

File: config/bin_version_strings.cfg [hunk 1]
```diff
@@ -42,6 +42,7 @@ comgt::"comgt\ version\ [0-9]\.[0-9]+"
 conntrack::"conntrack\ v[0-9]\.[0-9]\.[0-9]+\ \(conntrack-tools\)"
 conntrack::"conntrack\ v[0-9]\.[0-9]\.[0-9]\ \(conntrack-tools\)"
 coreutils::"\(GNU\ coreutils\)\ [0-9]\.[0-9]+$"
+chrony::"chrony[cd]\ \(chrony\)\ version [0-9]\.[0-9]"
```
Comment (m-1-k-3) [inline config/bin_version_strings.cfg:45]: could we do something like this?
chrony::"**^**chrony[cd]\ \(chrony\)\ version [0-9]\.[0-9]**$**"

Same on the other version identifiers

**Comment by m-1-k-3** [comment]
thank you @seanog8 for the new version identifiers. Probably we could start some identifier regexes with the beginning of the line "^" and end them with end of line "$"?

**Comment by seanog8** [comment]
I added it where it was possible. For some of them the line does not end with the version number.

File: config/bin_version_strings.cfg [hunk 2]
```diff
@@ -119,11 +124,12 @@ glibc::"GNU\ C\ Library\ \(.*\)\ stable\ release\ version\ [0-9]\.[0-9]+,\ "
 gnu_cpio::"\(GNU\ cpio\)\ [0-9]\.[0-9]+$"
 gnu_findutils::"\(GNU\ findutils\)\ [0-9]\.[0-9]+\.[0-9]+$"
 gnu_findutils::"\(GNU\ findutils\)\ [0-9]\.[0-9]+\.[0-9]+-[a-z]+$"
-gnu_gettext-runtime::"\(GNU\ gettext-runtime\)\ [0-9]\.[0-9]+\.[0-9]+\.[0-9]+$"
+gnu_gettext-runtime::"\(GNU\ gettext-runtime\)\ [0-9]\.[0-9]+\.[0-9]+"
```
Comment (m-1-k-3) [inline config/bin_version_strings.cfg:127]: as far as I can see we will loose the detection of the following version from the fritzbox firmware:
[+] Found version details:      gnu gettext-runtime     :       0.19.8.1        :       CVEs: 0         :       Exploits: 0  
could you please leave both in it and add a "$" to your new detection regex?

Comment (seanog8) [reply config/bin_version_strings.cfg]: done

**Comment by m-1-k-3** [comment]
looks good to me

**Review by m-1-k-3 — APPROVED** [review]

**Pull request merged**